v2/chunk.go

```diff
@@ -0,0 +1,35 @@
+package pie
+
+// Chunk returns slices with each length equalling chunkLength, except for the last slice which may contain less.
+// 	The order is maintained in each of the slices and slices themselves.
```

Comment:
This formatting is incorrect. The extra indent will cause this line to be formatted as code.

Reply:
ok. I will do it.

Reply:
done @elliotchance